locales/en/segments.yml

```diff
@@ -0,0 +1,105 @@
+    TransitBus: <strong>Bus</strong>
+    TransitBRT: <strong>Bus rapid service</strong>
+    TransitSchoolBus: <strong>Integrated school lines</strong>
+    TransitStreetCar: <strong>Tramway</strong> on street
+    TransitFerry: <strong>Ferry</strong>
+    TransitGondola: <strong>Gondola</strong>
+    TransitMonorail: <strong>Monorail</strong>
+    TransitRRT: <strong>Rapid rail transit</strong>, category A
+    TransitLRT: <strong>Light rail transit</strong>, category B
+    TransitLRRT: <strong>Light Rail Rapid Transit</strong>, category A
+    TransitHSR: <strong>High speed rail</strong>
+    TransitRegionalRail: <strong>Regional train</strong>
+    TransitOnDemand: <strong>On-demand transport</strong>
+    TransitTaxi: <strong>Taxibus</strong>
+    IntercityBus: <strong>Intercity bus</strong>
+    IntercityTrain: <strong>Intercity train</strong>
+    SchoolBus: <strong>School bus</strong> (yellow)
+    OtherBus: <strong>Chartered</strong> or private <strong>bus</strong>
+    Taxi: <strong>Taxi</strong> or equivalent (eg. Uber)
+    Paratransit: <strong>Paratransit</strong>
+    Wheelchair: <strong>Wheelchair</strong>
+    MobilityScooter: <strong>Mobility scooter</strong>
+    Motorcycle: <strong>Motorcycle</strong> (or motorized scooter/moped)
+    FerryWithCar: <strong>Ferry</strong> with vehicle
+    Plane: <strong>Airplane</strong>
+    OtherActiveMode: Other <strong>active mode</strong>
+    Other: <strong>Other</strong>
+    DontKnow: <strong>I don't know</strong> / Prefer not to answer
+    short:
+        walk: Walking
+        bicycle: Bicycle
+        bicycleElectric: E-Bike
+        scooterElectric: E-Scooter
+        bicycleBikeSharing: BIXI
+        bicycleBikeSharingElectric: BIXI E-Bike
+        bus: Bus
+        transitHeavy: Metro, REM or train
+        transitBus: Transit bus
+        intercityBus: Intercity bus
+        schoolBus: Yellow school bus
+        integratedSchoolLines: Integrated school lines
+        busOther: Chartered or private bus
+        transitTram: Tramway
+        transitSubway: Metro (subway)
+        transitREM: REM
+        transitRail: Commuter train
+        intercityRail: Intercity train
+        personalCar: Car driver with personal car
+        carDriver: Car driver
+        carPassenger: Car passenger
+        carDriverCarsharing: Carsharing
+        carDriverCarsharingStationBased: Station-based carsharing
+        carDriverCarsharingFreeFloating: Free floating carsharing
+        rentalCar: Rental car
+        transitTaxi: Taxibus / On-demand
+        taxi: Taxi
+        paratransit: Paratransit
+        plane: Airplane
+        otherActiveModes: Other active mode
+        motorcycle: Motorcycle or motorized scooter/moped
+        ferryNoCar: River shuttle or ferry without vehicle
+        ferryWithCar: Ferry with vehicle
+        other: Other
+        unknown: Unknown
+        dontKnow: Don't know
+ModeFirst: Which mode of transportation was used first?
```

Comment:
use the term "mode of transport" everywhere instead of mode of transportaiton or transportation mean/mean of transport/transportation